docs/data-sources/applications.md

```diff
@@ -0,0 +1,116 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "coolify_applications Data Source - coolify"
+subcategory: ""
+description: |-
+  Get a list of Coolify applications.
+---
+
+# coolify_applications (Data Source)
+
+Get a list of Coolify applications.
+
+
```

Comment:
The example usage is missing from the documentation, will need to run `make generate`